documentacion/webpay/README.md

```diff
@@ -605,7 +605,9 @@ import cl.transbank.webpay.WebpayOneClick;
 //...
 
 Configuration configuration = new Configuration();
-configuration.setCommerceCode("12345"); // acá va tu código de comercio
+
+// a continuación va tu código de comercio, Si el código que posees es de 8 dígitos debes anteponer 5970.
```

Comment:
I see this comment only on the Java snippet but not the other languages. Why?

Reply:
yes, my mistake, I already fix it :)